packages/core/src/renderer/extension-discovery/init.injectable.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import extensionDiscoveryInjectable from "../../extensions/extension-discovery/extension-discovery.injectable";
+import { beforeFrameStartsSecondInjectionToken } from "../before-frame-starts/tokens";
+import initializeExtensionLoaderInjectable from "../extension-loader/init.injectable";
+
+const initializeExtensionDiscoveryInjectable = getInjectable({
+  id: "initialize-extension-discovery",
+  instantiate: (di) => ({
+    id: "initialize-extension-discovery",
+    run: async () => {
+      const extensionDiscovery = di.inject(extensionDiscoveryInjectable);
+
+      await extensionDiscovery.init();
+    },
+    runAfter: di.inject(initializeExtensionLoaderInjectable),
+  }),
+  injectionToken: beforeFrameStartsSecondInjectionToken,
```

Comment:
How does one know to use `beforeFrameStartsSecondInjectionToken` over `beforeFrameStartsFirstInjectionToken`?

Reply:
If it depends on something that runs in one of the cluster/main frames ifs above in `packages/core/src/renderer/start-frame/start-frame.injectable.ts`

Reply:
Hopefully once we remove iframes this can be simplified.